introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb

```diff
@@ -0,0 +1,2425 @@
+   "execution_count": 22,
+   "id": "68945504",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Getting interpolated model predictions\n",
+      "working on 0 out of 100\n",
+      "working on 10 out of 100\n",
+      "working on 20 out of 100\n",
+      "working on 30 out of 100\n",
+      "working on 40 out of 100\n",
+      "working on 50 out of 100\n",
+      "working on 60 out of 100\n",
+      "working on 70 out of 100\n",
+      "working on 80 out of 100\n",
+      "working on 90 out of 100\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get interpolated model prediction\n",
+    "print (\"Getting interpolated model predictions\")\n",
+    "interpolated_model_preds, _ = calculate_accuracy(test_data, interpolated_model)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "0b46f81e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Old model accuracy 81.7%\n",
+      "New model accuracy 82.55%\n",
+      "Interpolated model accuracy 82.65%\n",
+      "Negative Flip Rate 3.25%\n",
+      "Negative Flip Rate Old and Interpolated (Ours) 2.225%\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get old model accuracy\n",
+    "old_acc = 100 * sum([old_pred == l for old_pred, l in zip(old_model_preds, labels)]) / float(len(labels))\n",
+    "print (f\"Old model accuracy {old_acc}%\")\n",
+    "\n",
+    "# Get new model accuracy\n",
+    "new_acc = 100 * sum([new_pred == l for new_pred, l in zip(new_model_preds, labels)]) / float(len(labels))\n",
+    "print (f\"New model accuracy {new_acc}%\")\n",
+    "\n",
+    "# Get interpolated model accuracy\n",
+    "interpolated_acc = 100 * sum([interpolated_pred == l for interpolated_pred, l in zip(interpolated_model_preds, labels)]) / float(len(labels))\n",
+    "print (f\"Interpolated model accuracy {interpolated_acc}%\")\n",
+    "\n",
+    "\n",
+    "# Calculate negative flip rate\n",
+    "nfr = 100 * sum([old_pred == l and new_pred != l for old_pred, new_pred, l in zip(old_model_preds, new_model_preds, labels)]) / float(len(labels))\n",
+    "print (f\"Negative Flip Rate {nfr}%\")\n",
+    "\n",
+    "# Calculate negative flip rate of old model and interpolated models\n",
+    "interpolate_nfr = 100 * sum([old_pred == l and interpolate_pred != l for old_pred, interpolate_pred, l in zip(old_model_preds, interpolated_model_preds, labels)]) / float(len(labels))\n",
+    "print (f\"Negative Flip Rate Old and Interpolated (Ours) {interpolate_nfr}%\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a933296",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4b3471b",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "conda_pytorch_p38",
```

Comment:
Hi Elman, you can change this line to '' "display_name": "Python 3 (PyTorch 1.8 Python 3.6 GPU Optimized)"